doc/release/upcoming_changes/25149.change.rst

```diff
@@ -0,0 +1,8 @@
+Out-of-bound axis not the same as ``axis=None``
+-----------------------------------------------
+In some cases ``axis=32`` or for concatenate any large value
+was the same as ``axis=None``.
+Except for ``concatenate`` this was deprecate.
```

Comment:
```suggestion
Except for ``concatenate`` this was deprecated in Numpy 1.23.
```

Was the deprecation noisy for other functions? Just wondering if we might want to merely deprecate this usage for `concatenate` since that was missed in 2022.